testsuite/features/secondary/srv_scc_user_credentials.feature

```diff
@@ -40,3 +56,7 @@ Feature: SCC user credentials in the Setup Wizard
     And I ask to delete the credentials for "SCC user"
     And I click on "Delete" in "Are you sure you want to delete these credentials?" modal
     Then I wait until I do not see "SCC user" text
+    And I wait for the trash icon to appear for "invalidname"
```

Comment:
When

Reply:
Shouldn't this also be changed in line 55 (not my code)?